src/index.js

```diff
@@ -140,6 +141,8 @@ app.post('/getToken', (request, response) => {
 if (process.env.ENVIRONMENT === 'production') {
   LinkService().detectLinks()
   ChatService().detectNewMessages()
+  // Update available superlinks everyday at midnight
+  schedule.scheduleJob('0 0 * * *', UserService().updateAvailableSuperlinks)
```

Comment:
midnight of where?

Reply:
Usa la del sistema, con lo cual habría que tener cuidado cuando corre en Heroku. Ahora me fijo si hay algo para que corra en otro timezone

Reply:
Estuve investigando bastante. Encontré [esta librería](https://www.npmjs.com/package/node-schedule-tz), pero la estuve probando y no funciona bien (Mejor dicho, no funciona). Tampoco tiene mucha documentación, asique no pude avanzar mucho. Dentro de la de `node-schedule` hay algunos PRs dando vuelta para agregar timezones pero ninguno cerrado. No me volvería extremadamente loco con esto, pero bueno.